Fortinet/fortigate/_meta/fields.yml

```diff
@@ -98,6 +98,37 @@ fortinet.fortigate.poluuid:
   name: fortinet.fortigate.poluuid
   type: keyword
 
+fortinet.fortigate.security_profile:
+  description: Name of the security profile used in the event
+  name: fortinet.fortigate.security_profile
+  type: keyword
+
+fortinet.fortigate.session.duration:
+  description: Duration in seconds between the last session log and the current session
+    log
+  name: fortinet.fortigate.session.duration
+  type: long
+
+fortinet.fortigate.session.received_bytes:
+  description: Increment of bytes which are received after the start of the session
+  name: fortinet.fortigate.session.received_bytes
+  type: long
+
+fortinet.fortigate.session.received_packets:
+  description: Increment of packets which are received after the start of the session
+  name: fortinet.fortigate.session.received_packets
+  type: long
+
+fortinet.fortigate.session.sent_bytes:
+  description: Increment of bytes which are sent after the start of the session
+  name: fortinet.fortigate.session.sent_bytes
+  type: long
+
+fortinet.fortigate.session.sent_packets:
+  description: Increment of packets which are sent after the start of the session
+  name: fortinet.fortigate.session.sent_packets
+  type: long
```

Comment:
Can you use ECS  `network.*`, `source.*`, `destination.*` and ` event.duration` fields for this information?